changes.d/533.fix.md

```diff
@@ -0,0 +1 @@
+Fixed bug introduced in 1.4.2 where job submission would fail if using a Cylc environment whose name does not match `cylc-8.X.Y` exactly.
```

Comment:
```suggestion
Fixed bug introduced in 1.4.2 where `cylc play` would fail if using a Cylc environment whose name does not match `cylc-8.X.Y` exactly.
```